.github/workflows/docker.yml

```diff
@@ -2,7 +2,6 @@ name: Docker
 
 on:
   push:
-    branches: [ main ]
```

Comment:
The aim was to run this workflow as a test on every push for now, until development has settled for a release. It could make sense to split of into two workflows then, one as a test which runs on every push/PR, one for publishing the image, which runs only when pushing to `master` (we should rename it to `main` indeed). What do you think?

Reply:
Ah lol, without defining branches, if tags are defined, it seems to not run on any push.

Reply:
you have to do a * in branches or remove tags

Reply:
Understood the conditions for registry and image upload now and changed them to happen only when a push is done to our `main` branch. So in every other situation is runs as test only.